LESSON-02/README.md

```diff
@@ -0,0 +1,189 @@
+# Introduction
+
+In this Lesson we explain the meaning of operators in programming, types
+and usages.
+
+# Lesson Index
+
+* 2 Operators
+* 2.1 Arithmetic Operators
+* 2.2 Comparison operators.
+* 2.3 Other operators.
+* 2.4 Order in operation.
+* 2.5 Testing operators.
+* 2.6 Exercises.
+* 2.7 Thoughts of an operator.
+
+# 2 Operators
+ 
+ Basically are used to perform operations on variables and values, sound as an easy thing
+ but it is a powerful tool inside programming. Most of the time, the needs for transforming date
+ are fulfilled by the combination of actions and operators in the code.
+
+## 2.1 Arithmetic operators.
+  In most programming languages arithmetic operators are used with numeric values to perform common arithmetical operations, such as addition, subtraction, multiplication etc.
+
+  Addition is represented by + symbol
+  
+    <?php 
+    $x = 10;  
+    $y = 6;
+    echo $x + $y;
+    ?>
+  
+  Will show 16 on the screen
+  	
+  Subtraction is represented by - symbol
+  
+    <?php 
+    $x = 10;  
+    $y = 6;
+    echo $x - $y;
+    ?>
+  
+  Will show 4 on the screen
+  
+  Multiplication is represented by - symbol
```

Comment:
Multiplication is represented by * symbol, no - symbol.

Reply:
Ok, really nice, you are paying attention!